docs/signed-apk-android.md

```diff
@@ -38,7 +38,8 @@ Navigate to that directory by using the command `cd /your/jdk/path` and use the
 ## Setting up Gradle variables
 
 1. Place the `my-upload-key.keystore` file under the `android/app` directory in your project folder.
-2. Edit the file `~/.gradle/gradle.properties` or `android/gradle.properties`, and add the following (replace `*****` with the correct keystore password, alias and key password),
+2. Edit the file `~/.gradle/gradle.properties` or `android/gradle.properties`, and add the following
+3. Replace `*****` with the correct keystore (line 3) and key (line 4) passwords, if you were not prompted to create a key password it will be the same as keystore password
```

Comment:
```suggestion
2. Edit the file `~/.gradle/gradle.properties` or `android/gradle.properties`, and add the following (replace `*****` with the correct keystore/key password. Those are the one you got asked to type with the `keytool` command before):
```

Reply:
This is pretty similar to what we had before, which was misleading. I wouldn't suggest doing such a small change, as it may not contribute to the goal of _saving people's time by writing clear instructions_.

Reply:
The current revision has two bullet points:

```
2. Edit the file `~/.gradle/gradle.properties` or `android/gradle.properties`, and add the following
3. Replace `*****` with the correct...
```

with a "the following" which is referred to the snippet below. So you should either reword it, merge it into a single point as suggested, or remove the ", and add the following" part.